src/Plugin/GraphQL/DataProducer/DataProducerProxy.php

```diff
@@ -135,8 +135,8 @@ public function cached($cached = TRUE) {
   /**
    * Resolve field value.
    *
-   * @param $value
-   * @param $args
+   * @param int $value
+   * @param string $args
```

Comment:
mixed

Reply:
reminding that `$args` should be `mixed` type